Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -63,7 +63,7 @@ import BraintreeDataCollector
     private var isConfigFromCache: Bool?
 
     /// Used for sending the type of flow, universal vs deeplink to FPTI
```

Comment:
Should we remove this comment?

Reply:
Ah, good catch, yes.

Reply:
Updated: https://github.com/braintree/braintree_ios/pull/1551/commits/e14f35ad23bc3a5e0a1f9e3658347f5ea389e2cd